mmv1/products/compute/FirewallPolicyRule.yaml

```diff
@@ -221,6 +234,27 @@ properties:
           Names of Network Threat Intelligence lists. The IPs in these lists will be matched against traffic source.
         item_type:
           type: String
+      - name: 'srcSecureTags'
+        type: Array
+        send_empty_value: true
```

Comment:
What is the reason to use `send_empty_value: true` here?

Reply:
@zli82016 I copied this 1:1 from the networkFirewallRules. The same feature was now introduced in hierarchical firewall rules as well. I was assuming this was because how the API currently works. We can try also without. wdyt?